civo/terraform/providers.tf

```
@@ -0,0 +1,18 @@
+terraform {
+  required_version = ">= 1.0.0"
+
+  required_providers {
+civo = {
```

nit: spacing on this line :)
```suggestion
    civo = {
```